src/show/pages/public/SignUp/index.js

```diff
@@ -1,9 +1,157 @@
-import React from 'react'
+import React, { useState } from 'react';
+import { Button, TextField, Split, Region } from '../../../components';
+import '../../../styles/style.css';
+import LeftImage from "../../../assets/img/image.jpg";
+import { CheckIcon, XMarkIcon } from '@heroicons/react/24/solid';
+
```

Comment:
the import should be proper and do this every where 

```
import React, { useState } from 'react';
import { CheckIcon, XMarkIcon } from '@heroicons/react/24/solid';

import { Button, TextField, Split, Region } from '../../../components';
import LeftImage from "../../../assets/img/image.jpg";

import '../../../styles/style.css';
```

Reply:
above you can top level is for absolute import i.e, from the packages,
after spacing 2nd level is for the component and anything which is global 
and the third is for same folder

Reply:
follow this in every file, it looks good and more readable

Reply:
done